Doc/library/exceptions.rst

```diff
@@ -90,8 +90,13 @@ The following exceptions are used mostly as base classes for other exceptions.
    .. method:: with_traceback(tb)
 
       This method sets *tb* as the new traceback for the exception and returns
-      the exception object.  It is usually used in exception handling code like
-      this::
+      the exception object.  It was more commonly used before the exception
+      chaining features of :pep:`484` became available.  The following example
+      shows how we can convert an instance of ``SomeException`` into an
+      instance of ``OtherException`` while preserving the traceback.  Once
+      raised, the current frame is pushed unto the traceback of the
```

Comment:
```suggestion
      raised, the current frame is pushed into the traceback of the
```

Guessing this should be `into` or `onto`?

Reply:
onto, thanks.